apps/browser/src/platform/browser/browser-api.ts

```diff
@@ -19,6 +19,15 @@ export class BrowserApi {
     return chrome.runtime.getManifest().manifest_version;
   }
 
+  /**
+   * Determines if the extension manifest version is the given version.
+   *
+   * @param expectedVersion - The expected manifest version to check against.
+   */
+  static isManifestVersion(expectedVersion: 2 | 3) {
```

Comment:
non-blocking naming nit (the nittiest) - don't bother if you don't need to make other changes, tho

```suggestion
  static isManifestVersion(expectedManifestVersion: 2 | 3) {
```